mitmproxy/addons/savehar.py

```diff
@@ -59,12 +66,12 @@ def flow_entry(self, flow: http.HTTPFlow, servers_seen: set[Server]) -> dict:
             ssl_time = -1.0
         elif flow.server_conn.timestamp_tcp_setup:
             assert flow.server_conn.timestamp_start
-            connect_time = 1000 * (
+            connect_time = (
```

Comment:
Why did we remove the `1000 *` here? IIRC the HAR spec was in milliseconds, or am I mixing up things?